smarts/core/road_map.py

```diff
@@ -143,8 +146,10 @@ def features(self) -> List[RoadMap.Feature]:
         def features_near(self, pose: Pose, radius: float) -> List[RoadMap.Feature]:
             raise NotImplementedError()
 
-        def shape(self, buffer_width: float = 0.0) -> Polygon:
-            """Returns a convex polygon, buffered by width (which must be non-negative), around this surface."""
+        def shape(self, width: float, buffer_width: float = 0.0) -> Polygon:
+            """Returns a convex polygon, buffered by width + buffered_width (which must be non-negative),
+            where buffer_width is the change in width around this surface (like 0.3 + width).
+            If you want to keep the width to original, pass original width and set buffer_width as 0"""
```

Comment:
I suggest:
```python
def shape(self, buffer_width: float = 0.0, default_width: Optional[float] = None) -> Polygon:
    """Returns a convex polygon representing this surface, buffered by buffered_width (which must be non-negative),
    where buffer_width is a buffer around the perimeter of the polygon.  In some situations, it may be desirable to 
    also specify a `default_width`, in which case the returned polygon should have a convex shape where the 
    distance across it is no less than buffered_width + default_width at any point."""
```

Note that the argument order is reversed and `default_width` is made optional.